extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -95,6 +95,7 @@ describes.realWin(
       resetServiceForTesting(win, 'consentStateManager');
       registerServiceBuilder(win, 'consentStateManager', function () {
         return Promise.resolve({
+          consentPageViewId64: () => 'consent_page_view_id_64',
```

Comment:
Let's change this into a value that looks more arbitrary so that future readers can more easily derive that it's mocked.
```suggestion
          consentPageViewId64: () => 'foo_consent_page_view_id_64_123',
```